./tests.verify: test performed on 2003.08.30 


#### Test: ./tests.verify running ./make_module_1.sh 
ERROR in ./tests.verify: execution failure arose from the command
  ./make_module_1.sh  >> /work/scripting/src/py/mixed/Grid2D/C/tests.v

CPU time of ./make_module_1.sh: 0.5 seconds on hpl-lapx30 i686, Linux


#### Test: ./tests.verify running ../Grid2Deff.py verify1
ERROR in ./tests.verify: execution failure arose from the command
  ../Grid2Deff.py verify1 >> /work/scripting/src/py/mixed/Grid2D/C/tests.v

CPU time of ../Grid2Deff.py: 0.2 seconds on hpl-lapx30 i686, Linux


#### Test: ./tests.verify running ../Grid2Deff.py exceptions1
ERROR in ./tests.verify: execution failure arose from the command
  ../Grid2Deff.py exceptions1 >> /work/scripting/src/py/mixed/Grid2D/C/tests.v

